package.json

```diff
@@ -121,6 +121,7 @@
   },
   "husky": {
     "hooks": {
+      "pre-commit": "lint-staged && yarn test:ci",
```

Comment:
see the next comment (remove `&& yarn test:ci`)

Reply:
Applied the changs but am receiving this error on commit.
```

× npm found some errors. Please fix them and try committing again.

> @hospitalrun/frontend@0.0.0-development test:ci C:\Users\55419\Code\hospitalrun-frontend
> cross-env CI=true react-scripts test "C:/Users/55419/Code/hospitalrun-frontend/.eslintrc.js"

No tests found, exiting with code 1
Run with `--passWithNoTests` to exit with code 0
In C:\Users\55419\Code\hospitalrun-frontend
82 files checked.
testMatch: C:/Users/55419/Code/hospitalrun-frontend/src/**/__tests__/**/*.{js,jsx,ts,tsx}, C:/Users/55419/Code/hospitalrun-frontend/src/**/*.{spec,test}.{js,jsx,ts,tsx} - 30 matches
testPathIgnorePatterns: \
ode_modules\\ - 82 matches
testRegex:  - 0 matches
Pattern: C:\\Users\\55419\\Code\\hospitalrun-frontend\\.eslintrc.js - 0 matches
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @hospitalrun/frontend@0.0.0-development test:ci: `cross-env CI=true react-scripts test "C:/Users/55419/Code/hospitalrun-frontend/.eslintrc.js"`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @hospitalrun/frontend@0.0.0-development test:ci script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\55419\AppData\Roaming
pm-cache\_logs\2020-02-07T13_30_53_752Z-debug.log
husky > pre-commit hook failed (add --no-verify to bypass
```

Reply:
I [pushed](https://github.com/HospitalRun/hospitalrun-frontend/pull/1780/commits/639f122b11e6eee2953d0ad40db6edbd22b82f61) changes myself and didn't have any trouble.